docs/ramalama-serve.1.md

```diff
@@ -187,6 +188,64 @@ CONTAINER ID  IMAGE                             COMMAND               CREATED
 3f64927f11a5  quay.io/ramalama/ramalama:latest  /usr/bin/ramalama...  17 seconds ago  Up 17 seconds  0.0.0.0:8082->8082/tcp  ramalama_YMPQvJxN97
 ```
 
+### Generate kserve service off of OCI Model car quay.io/ramalama/granite:1.0
+```
+$ ramalama serve --pull=never --threads 10 --port 8081 --generate kserve oci://quay.io/rhatdan/granite
+Generating kserve runtime file: granite-cuda-kserve-runtime.yaml
+Generating kserve file: granite-cuda-kserve.yaml
+
+$  cat granite-cuda-kserve-runtime.yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: llama.cpp-cuda-runtime
```

Comment:
Should this be granite instead of llama?

Reply:
That is the runtime. But maybe granite makes more sense.